packages/wallets/src/components/WalletImport.tsx

```diff
@@ -43,11 +63,47 @@ export default function WalletImport() {
     },
   });
 
-  const { fields } = useFieldArray({
+  const { fields, replace } = useFieldArray({
     control: methods.control,
     name: 'mnemonic',
   });
 
+  const submitMnemonicPaste = (mnemonicList: string) => {
+    closeMnemonicPaste();
+
+    let mList = mnemonicList.match(/\b(\w+)\b/g);
+    const intersection = mList.filter(element => options.includes(element));
```

Comment:
The type of `mList` is either an array or `null`.
For example, if you paste nothing and click import button on the dialog, `mnemonicList` will be `""`(empty string).
In that case, `mList` will be `null` because it will fail to `match` regexp and get an exception on the next line. (Trying to call `filter` on `null`)